code/modules/mob/living/simple_animal/bot/medbot.dm

```diff
@@ -280,16 +319,13 @@
 	if(H.stat == DEAD)
 		return
 
-	if((H == oldpatient) && (world.time < last_found + 200))
+	if((H.unique_datum_id == previous_patient) && (world.time < last_found + 200))
```

Comment:
```suggestion
	if((H.UID() == previous_patient) && (world.time < last_found + 200))
```
Nitpick, but `/datum/proc/UID()` returns `unique_datum_id` if it exists and generates one if not. I would rather stick to the intended API here.

Reply:
if you say so